workflows/cosign/versions/0.0.1/docs/sign.md

```diff
@@ -0,0 +1,29 @@
+# task
```

Comment:
you need to change the name of the file sign.md to cosign.md 
it should be as the template name 
also change the names of the files cosign.workfolw... and cosign.rbac.. just trim the cosign